test/bats/azure-test.bats

```diff
@@ -318,3 +318,47 @@ SLEEP_TIME=1
     result=$(kubectl get pod mutate-demo --namespace default -o json | jq -r ".spec.containers[0].image" | grep @sha)
     assert_mutate_success
 }
+
+@test "validate refresher reconcile count" {
```

Comment:
wonder if we can add a teardown method to reset the refreshInterval after the test is done to avoid throttling on AKV.

Reply:
In the third [test](https://github.com/ratify-project/ratify/blob/9622f6dfcd79f1a74aa193d20f288de180ba2fec/test/bats/azure-test.bats#L351) there is a teardown that deletes the KMP resource, which would only keep it alive until the final test finishes. However, I could add a clean up to each test and recreate the kmp within each test so it doesn't run longer than it needs to.

Reply:
QQ: does 👍  mean it's okay to delete it at the end in the final test or should I modify each test to create and delete the resource? :)

Reply:
In general, bats tests can be run independent of each other via the bats cli. We should try to keep tests independent and self-contained. Just my opinion.

Reply:
Roger roger (StarWars reference) I'll get cleanup logic for each test added.